src/components/post-card/PostCard.js

```diff
@@ -9,56 +9,99 @@ import './PostCard.scss';
 //  в котором будет флажок showComments
 //  и будет кнопка, лейба которой будет либо "show comments" либо "hide comments"
 
-export function PostCard(props) {
-  const { post, hasImage, comments = [], author } = props;
+export class PostCard extends React.Component {
 
-  console.log('hasImage', hasImage);
-  const { title, body } = post;
+    constructor(props) {
+        super(props);
 
-  const kittyUrl = `https://cataas.com/cat/says/hello%20world!?${Math.random() * 1000}`;
+        this.state = {
+            isOpen: false,
+            showComments: 'show comment'
+        }
+    }
 
-  const renderImage = () => {
-    return hasImage ? (
-      <img src={kittyUrl} />
-    ) : (
-      <img src={DefaultImg} />
-    );
-  };
+    onClick = () => {
+        console.log ('click');
+        this.state.isOpen ? this.setState( {
```

Comment:
Please take a look at your's colleague code and my comment and try to fix it in your code
https://github.com/AlyonaPianykh/react-may-2020/pull/166/files